CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+#### VERSION 2.0.0
```

Comment:
A suggestion: use the guidelines of the [keepachangelog](https://keepachangelog.com/en/1.0.0/).  
Separating the version into added/changed/removed makes the changes more structured.  
(Note: this is also `lein new app`'s default suggestion)

Reply:
Thanks!